src/Home.js

```diff
@@ -1,36 +1,36 @@
-import React, {Component} from "react";
-import './Home.css';
-class Home extends Component{
-    render(){
-        const arrow = "-------->"
-        return (
-          <div id="home" className="Home">
-            <div className="Home-welcome">Hello!</div>
-            <div className="Home-name">
-              <span>I'm </span>
-              <span className="Home-myName" style={{ color: "#A00" }}>
-                G
-              </span>
-              {"RISHMA".split("").map((ltr) => (
-                <div className="Home-myName">{ltr}</div>
-              ))}
-            </div>
-            <div className="Home-msg">Web Developer from India</div>
-            <div>
-              <a href="#projects">
-                <button className="Home-button"> View my Projects</button>
-              </a>
-              <a href="#about">
-                <button className="Home-button">About Me</button>
-              </a>
-            </div>
-            <div className="Home-scroll">
-              <div>SCROLL</div>
-              <div>{arrow}</div>
-            </div>
-          </div>
-        );
-    }
+import React, { Component } from "react";
+import "./Home.css";
+class Home extends Component {
+  render() {
+    const arrow = "-------->";
+    return (
+      <section id="home" className="Home">
+        <h1 className="Home-welcome">Hello!</h1>
```

Comment:
tip: Headings are like a book. The `h1` usually is the main title of the page, in this case, your name or your profession. Read more about [what makes good headings](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/)

Reply:
Do you recommend using `h1` for my name as well? 
Or using `p` tag for both Hello and name with change in font size using CSS? 
Earlier I had used `div` and changed font size using css.

Reply:
I answered in the comment above https://github.com/grishmakhanolkar/portfolio/pull/9/files#r1014625215 :)

To complement, try to avoid styling elements using their tag. Always prefer using classes because it will be easier to change the HTML without breaking the styles.